src/main/java/ua/softserveinc/tc/util/LocalDateUtil.java

```diff
@@ -13,9 +14,11 @@
 import java.util.Date;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-@Slf4j
 public final class LocalDateUtil {
 
+    @Log
```

Comment:
You may write this without lombok. 
Try this -  private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LocalDateUtil.class);